packages/realm_dart/test/realm_object_test.dart

```
@@ -522,7 +522,7 @@ void main() {
       subscription.cancel();
     });
 
-    test('empty list gives default subscriptions', () async {
+    test('empty list gives no subscriptions', () async {
```

Not sure if in this case we prefer to throw an exception or not